charms/worker/k8s/src/charm.py

```diff
@@ -206,8 +207,7 @@ def get_cluster_name(self) -> str:
         Returns:
             the cluster name.
         """
-        unit, node = self.unit.name, self.get_node_name()
-        if not self._stored.cluster_name:
```

Comment:
the type of any attribute of `StoredState` is now some kind of function call?  I don't think pylint is reading the type information correctly.  Anyway -- we're going to do a more strict check for empty string